packages/react-next/src/components/SpinButton/examples/SpinButton.Stateful.Example.tsx

```diff
@@ -42,18 +42,26 @@ const onSpinButtonValidate = (value: string) => {
   return String(value) + suffix;
 };
 
-export const SpinButtonStatefulExample: React.FC = () => (
-  <div style={{ width: '400px' }}>
-    <SpinButton
-      label={'SpinButton with custom implementation:'}
-      min={0}
-      max={100}
-      value={'7' + suffix}
-      onValidate={onSpinButtonValidate}
-      onIncrement={onSpinButtonIncrement}
-      onDecrement={onSpinButtonDecrement}
-      incrementButtonAriaLabel={'Increase value by 2'}
-      decrementButtonAriaLabel={'Decrease value by 2'}
-    />
-  </div>
-);
+export const SpinButtonStatefulExample: React.FC = () => {
+  const [spinButtonValue, setSpinButtonValue] = React.useState<string>('7' + suffix);
```

Comment:
please add a unit test which covers controlled case so we don't regress this feature again.